gempak/ush/gfs_meta_opc_np_ver

```diff
@@ -148,91 +148,91 @@ for verday in ${verdays}
     cominday=`echo ${verday} | cut -c -8`
     #XXW export HPCGFS=$COMROOT/nawips/prod/${mdl}.${cominday}
     # BV export HPCGFS=$COMROOT/nawips/${envir}/${mdl}.${cominday}
-    export HPCGFS=${COMINgempak}/${mdl}.${cominday}/${cyc}/gempak
+    export HPCGFS=${COMINgempak}/${mdl}.${cominday}/${cyc}/atmos/gempak
```

Comment:
"export COMPONENT=${COMPONENT:-atmos} " should be defined in all gempak scripts and replaced the hardwired "atmos" with "$COMPONENT"

Reply:
Will fix, thanks!

Reply:
Fixed.